test suite test_generate fails on aarch64, ppc64le and s390x (musl libc)

Running the test_generate fails on aarch64, ppc64le and s390x on Alpine Linux. It appears to pass on x86_64, x86 (32bit)f.

````
Test project /home/ncopa/aports/community/lastpass-cli/src/lastpass-cli-1.2.1/build
      Start  1: test_login
 1/18 Test  #1: test_login ........................   Passed    0.10 sec
      Start  2: test_login_wrong_pw_should_fail
 2/18 Test  #2: test_login_wrong_pw_should_fail ...   Passed    0.07 sec
      Start  3: test_add_account
 3/18 Test  #3: test_add_account ..................   Passed    0.16 sec
      Start  4: test_add_note
 4/18 Test  #4: test_add_note .....................   Passed    0.15 sec
      Start  5: test_add_note_with_notes
 5/18 Test  #5: test_add_note_with_notes ..........   Passed    0.15 sec
      Start  6: test_add_ssn_name
 6/18 Test  #6: test_add_ssn_name .................   Passed    0.15 sec
      Start  7: test_add_ssh_key
 7/18 Test  #7: test_add_ssh_key ..................   Passed    0.24 sec
      Start  8: test_edit_username
 8/18 Test  #8: test_edit_username ................   Passed    0.15 sec
      Start  9: test_edit_field
 9/18 Test  #9: test_edit_field ...................   Passed    0.15 sec
      Start 10: test_edit_reprompt
10/18 Test #10: test_edit_reprompt ................   Passed    0.23 sec
      Start 11: test_duplicate
11/18 Test #11: test_duplicate ....................   Passed    0.15 sec
      Start 12: test_generate
12/18 Test #12: test_generate .....................***Failed    0.11 sec
      Start 13: test_show
13/18 Test #13: test_show .........................   Passed    0.13 sec
      Start 14: test_show_note
14/18 Test #14: test_show_note ....................   Passed    0.12 sec
      Start 15: test_show_reprompt
15/18 Test #15: test_show_reprompt ................   Passed    0.14 sec
      Start 16: test_ls
16/18 Test #16: test_ls ...........................   Passed    0.12 sec
      Start 17: test_export
17/18 Test #17: test_export .......................   Passed    0.14 sec
      Start 18: test_export_extended
18/18 Test #18: test_export_extended ..............   Passed    0.14 sec

94% tests passed, 1 tests failed out of 18

Total Test time (real) =   2.63 sec

The following tests FAILED:
         12 - test_generate (Failed)
Errors while running CTest
make[1]: *** [Makefile:74: test] Error 8
make[1]: Leaving directory '/home/ncopa/aports/community/lastpass-cli/src/lastpass-cli-1.2.1/build'
make: *** [Makefile:27: test] Error 2
````

This gcc warning explains the problem:
````
/home/ncopa/aports/community/lastpass-cli/src/lastpass-cli-1.2.1/cmd-generate.c:77:78: warning: comparison is always true due to limited range of data type [-Wtype-limits]
  while ((option = getopt_long(argc, argv, "c", long_options, &option_index)) != -1) {
                                                                              ^~
````

`option` is a char and `getopt_long` returns int. It is a bit weird that it does not fail on x86_64 and x86.

--- a/cmd-generate.c
+++ b/cmd-generate.c
@@ -61,7 +61,7 @@ int cmd_generate(int argc, char **argv)
 		{"clip", no_argument, NULL, 'c'},
 		{0, 0, 0, 0}
 	};
-	char option;
+	int option;
 	int option_index;
 	char *username = NULL;
 	char *url = NULL;

